README.md

```diff
@@ -33,14 +36,15 @@ We will cover pre-trained AI APIs, such as [computer vision](https://azure.micro
 * **23 mins** - Build this sample into an app using Logic apps *(attendees - try it yourself)* [Go to Section](#build-custom-ai-into-an-application---azure-logic-apps)
 * **2 mins** - Useful links to share and finish
 
-
 ## Content Walk-through
 
 ## Microsoft Azure Cognitive Services - Computer Vision and Text Analytics
 
-First show the Microsoft Azure Cognitive Services in their simplest format - within the web browser. 
+First show the Microsoft Azure Cognitive Services in their simplest format - within the web browser.
 **Navigate to [https://azure.microsoft.com/en-gb/services/cognitive-services/](https://azure.microsoft.com/en-gb/services/cognitive-services/?WT.mc_id=ainights-github-amynic)**
 
+To ensure fluidity in your presentation, you can have the page opened in your browser before your session beginsand type in **Azure Cognitive Services** in the search box; it will usually be the first result displayed.
```

Comment:
double check spelling and spacing before we merge

Reply:
Yup will get this done